app/views/photos/_form.html.erb

```diff
@@ -1,42 +1,29 @@
 <%= form_with(model: photo) do |form| %>
   <% if photo.errors.any? %>
     <div id="error_explanation">
-      <h2><%= pluralize(photo.errors.count, "error") %> prohibited this photo from being saved:</h2>
-
-      <ul>
+      <ul class="list-unstyled">
         <% photo.errors.each do |error| %>
-          <li><%= error.full_message %></li>
+          <li>
+            <div class="alert alert-danger" role="alert">
+              <%= error.full_message %>
+            </div>
+          </li>
```

Comment:
A further refactoring could involve using [`full_messages_for`](https://apidock.com/rails/v5.2.3/ActiveModel/Errors/full_messages_for) for each attribute.